### ISSUE TITLE
 Module DateTime.DateTime, line 702, in _parse_args triggered through members last_login = ''

### ISSUE THREAD

**Issue by zopyx** [plone/Products.CMFPlone#3656]
Plone 6.0.0b3 with members imported through collective.exportimport:

For some member logins, we see the following error:

```

2022-10-17 16:55:43,945 ERROR   [Zope.SiteErrorLog:35][waitress-0] SyntaxError: http://localhost:42080/eteaching/login
Traceback (innermost last):
  Module ZPublisher.WSGIPublisher, line 167, in transaction_pubevents
  Module ZPublisher.WSGIPublisher, line 376, in publish_module
  Module ZPublisher.WSGIPublisher, line 271, in publish
  Module ZPublisher.mapply, line 85, in mapply
  Module ZPublisher.WSGIPublisher, line 68, in call_object
  Module z3c.form.form, line 233, in __call__
  Module z3c.form.form, line 228, in update
  Module plone.app.z3cform.csrf, line 21, in execute
  Module z3c.form.action, line 98, in execute
  Module z3c.form.button, line 301, in __call__
  Module z3c.form.button, line 159, in __call__
  Module Products.CMFPlone.browser.login.login, line 179, in handleLogin
  Module Products.CMFPlone.browser.login.login, line 140, in _post_login
  Module DateTime.DateTime, line 447, in __init__
  Module DateTime.DateTime, line 702, in _parse_args
SyntaxError: <no detail available>
```

This is obviously trigger with `login_time=''`:

```

(Pdb) pp arg
''
(Pdb) pp args
('',)
(Pdb) up
> /Users/ajung/src/plone6.buildout/eggs/DateTime-4.4-py3.9.egg/DateTime/DateTime.py(447)__init__()
-> return self._parse_args(*args, **kw)
(Pdb) up
> /Users/ajung/src/plone6.buildout/eggs/Products.CMFPlone-6.0.0b3-py3.9.egg/Products/CMFPlone/browser/login/login.py(140)_post_login()
-> login_time = DateTime(login_time)
(Pdb) pp login_time
''
```

**Comment by davisagli** [comment]
@zopyx Looking at the existing code, `login_time` needs to either be a DateTime or a string that can be used to construct a DateTime with a single arg. Empty string is not supported. So this sounds like it might be a bug in collective.exportimport, or in your data.

You can set login_time to DateTime('2000/01/01') if you want to consider the user's next login as their initial login: https://github.com/plone/plone.login/blob/404b457e227633b834ac8d702e1dc4f11977c26b/src/plone/login/browser/login.py#L162

I'll leave this open since the code in the traceback could be more robust and at least raise a more helpful exception when the input is invalid.

**Comment by mauritsvanrees** [comment]
I have included a fix for this in PR #3714 (Plone 5.2) and PR #3715 (Plone 6.0). My fix is to log a warning, and conclude that the user has not logged in yet, which is the only thing this part of the code needs to do.

Note that the PRs mainly target a different problem, but it touches the same code, so it made sense for me to combine the fixes.
